source/configuration_manual/fts/flatcurve.rst

```diff
@@ -0,0 +1,168 @@
+.. _fts_backend_flatcurve:
+
+Flatcurve FTS Engine
+====================
+
+.. versionadded:: 2.4.0;3.0.0
```

Comment:
3.0.0 needs to be Pro 3.0.  (There might be an option general discussion about this still)

Reply:
Thinking about this more, this should exclusively be "2.4.0".  flatcurve will not be shipped with Pro 3.0, so it is not correct to list 3.0.0 in it.  (Features not supported in Pro starting in 3.0 will not be available/configurable there).